projects/isy-angular-widgets/src/lib/incomplete-date/incomplete-date.component.ts

```diff
@@ -142,23 +149,20 @@ export class IncompleteDateComponent implements ControlValueAccessor, Validator,
       const dateStr = [partDay, partMonth, partYear].join('.');
       input.value = this.inputValue = dateStr;
 
-      // Mouse position (specific fixed numbers required)
-      /* eslint-disable @typescript-eslint/no-magic-numbers */
       switch (inputMousePosition) {
-        case 0:
-        case 3:
-          inputMousePosition += 3;
+        case cursorPositionAtFirstDayLetter:
```

Comment:
the missing break in this case seems prone to errors?

Reply:
There are breaks after each case block